### PR TITLE
bottom-sheet: tweak UI + reorganize

### PR DESCRIPTION
**Pull request by saghul**
- re-add thepand icon, shaped like a pill
- round top corners

### CODE REVIEW

**Comment by saghul** [comment]
Screen Shot 2020-03-23 at 10 12 58

**Comment by saghul** [comment]
New menu order.

Screen Shot 2020-03-24 at 12 24 14
Screen Shot 2020-03-24 at 12 24 06

File: react/features/invite/components/add-people-dialog/native/InviteButton.js [hunk 1]
```diff
@@ -36,7 +42,13 @@ class InviteButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        this.props.dispatch(setAddPeopleDialogVisible(true));
+        const { _addPeopleEnabled, dispatch } = this.props;
```
Comment (zbettenbuk) [inline react/features/invite/components/add-people-dialog/native/InviteButton.js]: This looks the same code as my lonely experience share button code. Don't we want to extract it to a function?

File: react/features/toolbox/components/native/OverflowMenu.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 import React, { PureComponent } from 'react';
-import { Platform } from 'react-native';
+import { Platform, TouchableOpacity, View } from 'react-native';
 import Collapsible from 'react-native-collapsible';
 
 import { ColorSchemeRegistry } from '../../../base/color-scheme';
 import { BottomSheet, hideDialog, isDialogOpen } from '../../../base/dialog';
-import { CHAT_ENABLED, IOS_RECORDING_ENABLED, getFeatureFlag } from '../../../base/flags';
+import { IconDragHandle } from '../../../base/icons';
```
Comment (zbettenbuk) [inline react/features/toolbox/components/native/OverflowMenu.js]: Alphabetical ordering seems to be wrong here

**Pull request merged**